lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -266,6 +269,12 @@ class TripBasicsPane extends Component<TripBasicsProps> {
                 )
               })}
             </AvailableDays>
+            <Checkbox checked={isOneTime} onChange={this._handleOneTimeTrip}>
+              <FormattedMessage
+                id="components.TripBasicsPane.onlyOnDate"
+                values={{ date: itinerary.startTime }}
+              />
+            </Checkbox>
```

Comment:
Should this maybe be a radio button instead? I'm thinking of having a big toggle switch between "monitor once" and "monitor regularly" and that way we can hide the calendar view if we have to

Reply:
Using radio buttons in https://github.com/opentripplanner/otp-react-redux/pull/1186/commits/29567d46154d705ed278aef20fb24748f3da24e2. I added the hide thing in https://github.com/opentripplanner/otp-react-redux/pull/1186/commits/d39120739f345b5586bec26377e9e4cafe17ab27. I think that makes things clearer.